src/NuGet.Services.BasicSearch/Web.config

```diff
@@ -66,6 +66,10 @@
   </system.webServer>
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+			<dependentAssembly>
```

Comment:
nit: consistent indentation.

might be worth deleting this whole `<runtime>` section and regenerating it from scratch using that build warning in VS.

Reply:
Fixed